src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4919,28 +4919,41 @@ namespace ZoneEquipmentManager {
                     !DataGlobals::WarmupFlag && !DataGlobals::DoingSizing && !FirstHVACIteration) {
                     if (!thisZoneEquip.FlowError) {
```

Comment:
Just a side note here. !DataGlobals::ZoneSizingCalc && !DataGlobals::SysSizingCalc are both set inside ManageSizing() while !DataGlobals::DoingSizing is set outside that function. Could remove ZoneSizingCalc and SysSizingCalc from this IF test.

        DoingSizing = true;
        ManageSizing();

        BeginFullSimFlag = true;
        SimsDone = false;
        if (DoDesDaySim || DoWeathSim || DoHVACSizingSimulation) {
            DoOutputReporting = true;
        }
        DoingSizing = false;

Reply:
@rraustad Thanks. I will clean that up and check elsewhere.

Reply:
I didn't check but I wonder if isPulseZoneSizing is complete before BeginFullSimFlag = true is set? If so, could reduce this even futher.

Reply:
extern bool isPulseZoneSizing;                 // true during the set of zone sizing calcs that include the "pulse" for the load component report

Reply:
Forget BeginFullSimFlag, it's set and reset multiple times.

Reply:
Likewise, `isPulseZoneSizing` is only true inside ManageSizing, so `DoingSizing` covers that.